.gitignore

```diff
@@ -1,7 +1,7 @@
 .npmrc
```

Comment:
Your branch is too outdated with master, please help merge master to this branch